doc/tutorial/introduction/introduction_align.cpp

```diff
@@ -1,17 +1,13 @@
-#include <seqan3/io/sequence_file/output.hpp>
-
-//! [sequence_input_include]
-#include <seqan3/core/debug_stream.hpp>      // for debug_stream
-#include <seqan3/io/sequence_file/input.hpp> // for sequence_file_input
-//! [sequence_input_include]
-
-//! [alignment_include]
-#include <tuple>                                                          // for std::make_pair
-#include <seqan3/alignment/aligned_sequence/aligned_sequence_concept.hpp> // for alignment stream operator
-#include <seqan3/alignment/pairwise/align_pairwise.hpp>                   // for align_pairwise
-//! [alignment_include]
-
-using seqan3::operator""_dna4;
+//! [alignment]
+#include <seqan3/std/filesystem>                                            // for tmp_dir
+#include <vector>                                                           // for std::vector
+#include <seqan3/alignment/aligned_sequence/aligned_sequence_concept.hpp>   // for alignment stream operator
+#include <seqan3/alignment/pairwise/align_pairwise.hpp>                     // for align_pairwise
```

Comment:
sort the includes and distinguish between stl headers and seqan headers by a new line